FFMpegCore/FFMPEG/Pipes/RawVideoPipeDataWriter.cs

```diff
@@ -0,0 +1,70 @@
+﻿using FFMpegCore.FFMPEG.Argument;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace FFMpegCore.FFMPEG.Pipes
+{
+    /// <summary>
+    /// Implementation of <see cref="IPipeDataWriter"/> for a raw video stream that is gathered from <see cref="IEnumerator{IVideoFrame}"/> 
+    /// </summary>
+    public class RawVideoPipeDataWriter : IPipeDataWriter
+    {
+        public string StreamFormat { get; set; }
+        public int Width { get; set; }
+        public int Height { get; set; }
+        public int FrameRate { get; set; } = 25;
+        private IEnumerator<IVideoFrame> framesEnumerator;
```

Comment:
Why IEnumerator instead of IEnumerable?

Reply:
Because i'm using first frame in `GetFormat()` method

Reply:
Cool. I just wanted to hear if there was a reason for not using the very commin IEnumerable